packages/central-server/src/apiV2/import/importSurveys/importSurveyQuestions.js

```diff
@@ -23,10 +23,16 @@ import {
   SURVEY_METADATA,
 } from './processSurveyMetadata';
 import { caseAndSpaceInsensitiveEquals, convertCellToJson } from './utilities';
+import { RECORDS } from '@tupaia/database';
 
 const QUESTION_TYPE_LIST = Object.values(ANSWER_TYPES);
 const VIS_CRITERIA_CONJUNCTION = '_conjunction';
 
+const objectsAreEqual = (a, b) => {
+  if (!!a === !!b) return true;
+  return JSON.stringify(a) === JSON.stringify(b);
+};
```

Comment:
I don't think this function is always going to be correct. I would probably just outsource this to [lodash isEqual](https://lodash.com/docs/4.17.15#isEqual) here since we need to do a deep equality check?! Or if you update the function it would probably be good to add a unit test for it 

<img width="466" alt="Screenshot 2024-09-12 at 3 23 58 PM" src="https://github.com/user-attachments/assets/f8fbc9a3-cddf-41ea-ac4c-ebb7cf711ba0">